README.md

```diff
@@ -159,7 +161,8 @@ This action should be triggered after adding or updating any templates and befor
 > **Note:** If you only need to update an existing template, you can simply modify the corresponding `.hbs` file and trigger the **Sync Handlebars Template with SendGrid** action. There is no need to redeploy or rebuild Docker images in this case.
 
 ## Environment Variables
-We do not have a committed `.env` file as it goes against best practice, per https://www.npmjs.com/package/dotenv. However, we do provide a sample `.env.sample` file that can be copied into a `.env` file to get development up and running. 
```

Comment:
This is autoformatting adding this line, not relevant to this but is good to have.